scene/gui/tree.cpp

```diff
@@ -3633,12 +3681,74 @@ void Tree::gui_input(const Ref<InputEvent> &p_event) {
 					mpos.y += v_scroll->get_value();
 				}
 
-				TreeItem *old_it = cache.hover_item;
-				int old_col = cache.hover_cell;
-
 				int col, h, section;
 				TreeItem *it = _find_item_at_pos(root, mpos, col, h, section);
 
+				// Find eventual hovered button in cell.
```

Comment:
```suggestion
				// Find possible hovered button in cell.
```
Eventual = sooner or later